pkg/fatfs/Makefile.include

```diff
@@ -0,0 +1,2 @@
+INCLUDES += -I$(BINDIRBASE)/pkg/$(BOARD)
+DIRS += $(BINDIRBASE)/pkg/$(BOARD)/fatfs
```

Comment:
Compile of fatfs fails on macOS with:
```
/RIOT/tests/pkg_fatfs/bin/pkg/native/fatfs/ff.c:4942:51: error: for loop has empty body
      [-Werror,-Wempty-body]
        for (slen = 0; (UINT)label[slen] >= ' '; slen++) ;      /* Get name length */
                                                         ^
/RIOT/tests/pkg_fatfs/bin/pkg/native/fatfs/ff.c:4942:51: note: put the semicolon on a
      separate line to silence this warning
```

please add the following to compile on macOS:
```
ifeq ($(shell uname -s),Darwin)
    CFLAGS += -Wno-empty-body
endif
```